docs/reference-guides/theme-json/theme-json-reference-v1.md

```diff
@@ -0,0 +1,142 @@
+# Version 1 Reference
+
+Theme.json version 2 has been released, see the [theme.json migration guide](/docs/reference-guides/theme-json/theme-json-migrations.md#migrating-from-v1-to-v2) for updating to the latest version.
+
+<!-- START TOKEN Autogenerated - DO NOT EDIT -->
+## Settings
+
+
+### border
+
+Settings related to borders.
+
+| Property  | Type   | Default | Props  |
+| ---       | ---    | ---    |---   |
+| customColor | boolean | false |  |
+| customRadius | boolean | false |  |
+| customStyle | boolean | false |  |
+| customWidth | boolean | false |  |
+
+---
+
+### color
+
+Settings related to colors.
+
+| Property  | Type   | Default | Props  |
+| ---       | ---    | ---    |---   |
+| background | boolean | true |  |
+| custom | boolean | true |  |
+| customDuotone | boolean | true |  |
+| customGradient | boolean | true |  |
+| duotone | array |  | colors, name, slug |
+| gradients | array |  | gradient, name, slug |
+| link | boolean | false |  |
+| palette | array |  | color, name, slug |
+
+---
+
+### layout
+
+Settings related to layout.
+
+| Property  | Type   | Default | Props  |
+| ---       | ---    | ---    |---   |
+| contentSize | string |  |  |
+| wideSize | string |  |  |
+
+---
+
+### spacing
+
+Settings related to spacing.
+
+| Property  | Type   | Default | Props  |
+| ---       | ---    | ---    |---   |
+| customMargin | boolean | false |  |
+| customPadding | boolean | false |  |
+| units | array | px,em,rem,vh,vw,% |  |
+
+---
+
+### typography
```

Comment:
In [v1](https://github.com/WordPress/wordpress-develop/blob/5.8/src/wp-includes/class-wp-theme-json.php#L181), we only supported `customFontSize`, `customLineHeight`, `dropCap`, and `fontSizes`.